test/manifold_test.cpp

```diff
@@ -86,43 +67,39 @@ TEST(Manifold, ValidInput) {
 }
 
 TEST(Manifold, InvalidInput1) {
-  Mesh in = Tet();
-  in.vertPos[2][1] = NAN;
+  MeshGL in = TetGL();
+  in.vertProperties[2 * 3 + 1] = NAN;
   Manifold tet(in);
   EXPECT_TRUE(tet.IsEmpty());
   EXPECT_EQ(tet.Status(), Manifold::Error::NonFiniteVertex);
 }
 
 TEST(Manifold, InvalidInput2) {
-  Mesh in = Tet();
-  std::swap(in.triVerts[2][1], in.triVerts[2][2]);
+  MeshGL in = TetGL();
+  std::swap(in.triVerts[2 * 3 + 1], in.triVerts[2 * 3 + 2]);
   Manifold tet(in);
   EXPECT_TRUE(tet.IsEmpty());
   EXPECT_EQ(tet.Status(), Manifold::Error::NotManifold);
 }
 
 TEST(Manifold, InvalidInput3) {
-  Mesh in = Tet();
-  for (ivec3& tri : in.triVerts) {
-    for (int i : {0, 1, 2}) {
-      if (tri[i] == 2) tri[i] = -2;
-    }
+  MeshGL in = TetGL();
+  for (uint32_t& triVert : in.triVerts) {
+    if (triVert == 2) triVert = -2;
   }
   Manifold tet(in);
   EXPECT_TRUE(tet.IsEmpty());
   EXPECT_EQ(tet.Status(), Manifold::Error::VertexOutOfBounds);
 }
 
 TEST(Manifold, InvalidInput4) {
-  Mesh in = Tet();
-  for (ivec3& tri : in.triVerts) {
-    for (int i : {0, 1, 2}) {
-      if (tri[i] == 2) tri[i] = 4;
-    }
+  MeshGL in = TetGL();
+  for (uint32_t& triVert : in.triVerts) {
+    if (triVert == 2) triVert = 4;
   }
   Manifold tet(in);
   EXPECT_TRUE(tet.IsEmpty());
-  EXPECT_EQ(tet.Status(), Manifold::Error::VertexOutOfBounds);
+  EXPECT_EQ(tet.Status(), Manifold::Error::NotManifold);
```

Comment:
This is surprising - the tetrahedron only has 4 verts, so this is clearly out of bounds. Are we checking for errors in a different order now?

Reply:
`TetGL` has duplicated vertices. Or should we change this test to test vertex out of bounds?

Reply:
Ah, apparently we already have this check in `InvalidInput7`. And I should probably rename that to `InvalidInput6`.

Reply:
Oh, I see now - let's just update the test to change that index to 7 instead of 4, since this one has more verts.

Reply:
we already have InvalidInput6